_gitbook/conventions/README.md

```diff
@@ -0,0 +1,3 @@
+# Conventions
+
+If you follow these conventions, your code will be more accessible to other developers. If you use a standard [coding style](coding_style.md), your project will be navigable and readable to others. If you write [documentation](documenting_code.md), others will understand the intent and behavior of your code.
```

Comment:
Thanks for you suggestions, how's this?

https://github.com/rmosolgo/crystal/commit/9d6f6d1cabc1589cbc4948d0c9ff07a14d4f9d74